student.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :id, :classroom
+
+  def initialize(age, classroom, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @id = Random.rand(1..1000)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please remove `id` from this class. `id` is already defined as an instance variable in the parent class that is also available for children's classes.